terraform/api_gateway.tf.json.template.py

```diff
@@ -206,6 +206,14 @@ def for_domain(cls, domain):
                 }
             }
         },
+        *([{
+            'aws_lambda_function_event_invoke_config': {
+                'forward_alb_logs': {
+                    'function_name': '${aws_lambda_function.indexer_forward_alb_logs.function_name}',
+                    'maximum_retry_attempts': 0
+                }
+            }
+        }] if config.enable_log_forwarding else []),
```

Comment:
```suggestion
        }] 
        if config.enable_log_forwarding else 
        []),
```

Plenty of precedent for this, too.